modules/directives/src/main/scala/scala/build/preprocessing/directives/StrictDirective.scala

```diff
@@ -6,6 +6,7 @@ case class StrictDirective(
   key: String,
   values: Seq[Value[_]]
 ) {
+  override def toString: String = s"//> $key \"${values.mkString("\",  \"")}\""
```

Comment:
Can we assume that each directive has at least one value for? Here we should check that the list of values is not empty and not add quota before?

Reply:
Good catch, should be fixed